.github/workflows/flow-build-artifacts.yml

```diff
@@ -38,21 +38,23 @@ on:
         default: all
 
 jobs:
+  get-latest-redis-tag:
```

Comment:
Can we put this inside the `steps` of the `setup` step?

Reply:
`setup` has it in `needs` - so it is clear it needs to do `get-latest-redis-tag`
And it can can be reused.
If we "inline" it as a step - will not have a dedicated name

Reply:
Yeah I saw the `needs`.
Since the `get-redis` variable is exposed from the `setup` step it seems cleaner to me.

Reply:
IMO, the current implementation is nice, but we can also fix the redis version to a specific patch version and remove the need to check for the latest release, as we don't really care about the redis version as long as it's compatible with the module version.
We don't use it for tests, only for loading the module and getting the `COMMANDS` info

Reply:
Isn't it what we currently have? A specific job that will get the redis version compatible with the module version? The ability to also take the latest patch from the compatible version is not a problem, is it?

Reply:
I'm saying we can pin the redis version to 7.4.3 hard-coded, without any job

Reply:
It's not a problem, but it also not necessary

Reply:
I am also fine with latest 7.4.x. Eventually, the packed module would also be loaded with and run against newer versions than 7.4.3.

Reply:
True, but in this workflow, it doesn't matter. We should **test** the module with the latest compatible version, but in the pack workflow we shouldn't care

Reply:
Yeah we can do that as well I see no harm
Let's just keep the ref-setting logic in the `setup` step

Reply:
It shouldn't be possible to call another workflow from within a step, and we also shouldn't duplicate the logic of `get-latest-tag`.
We should do one of the two:
1. Use `get-latest-tag`, external to `setup`
2. Use some specific and acceptable redis tag

Reply:
Current state fails https://github.com/RediSearch/RediSearch/actions/runs/14957133919

Reply:
That's what I was worried about..
Both options sound good to me 👍